specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -900,6 +900,191 @@
+        "description": "Deletes the blob inventory policy associated with the specified storage account.",
+        "x-ms-examples": {
+          "StorageAccountDeleteBlobInventoryPolicy": {
+            "$ref": "./examples/StorageAccountDeleteBlobInventoryPolicy.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/BlobInventoryPolicyName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Delete the blob inventory policy successfully."
+          },
+          "204": {
+            "description": "No Content -- The blob inventory policy does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/inventoryPolicies": {
+      "get": {
+        "tags": [
+          "BlobInventoryPolicies"
+        ],
+        "operationId": "BlobInventoryPolicies_List",
+        "description": "Gets the blob inventory policy associated with the specified storage account.",
+        "x-ms-examples": {
+          "StorageAccountGetBlobInventoryPolicy": {
+            "$ref": "./examples/StorageAccountListBlobInventoryPolicy.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Get the blob inventory policies successfully.",
+            "schema": {
+              "$ref": "#/definitions/ListBlobInventoryPolicy"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
```

Comment:
>null [](start = 26, length = 4)

Will you need paging for this in the foreseeable future? If yes, adding paging after the fact is a breaking change, but it is safe to set this to "nextLink" and have your backend always return 1 page until you decide to support paging. See [https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi009-always-add-x-ms-pageable-to-list-calls](https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi009-always-add-x-ms-pageable-to-list-calls) for more details.

Reply:
We Will Never have more than 1 blob inventory policy per storage account, it cannot be supported instead we have multiple rules within a single policy at the account level, even in the stable release version. This is a singleton resource and let's keep it like this in the interest of time. :)

Reply:
Sounds good.